.travis.yml

```diff
@@ -68,7 +68,32 @@ jobs:
         - GOARM=7 GOARCH=arm GOOS=linux CGO_ENABLED=0 make itest-only
       arch: arm64
 
+    - name: Btcd Integration Windows
+      script:
+        - make itest-windows
+      os: windows
+      before_install:
+        - choco upgrade --no-progress -y make netcat curl findutils
+        - export MAKE=mingw32-make
+      after_script:
+        - |-
+          case $TRAVIS_OS_NAME in
+            windows)
+              echo "Uploading to termbin.com..."
+              for f in ./lntest/itest/*.log; do cat $f | nc termbin.com 9999 | xargs -r0 printf "$f"' uploaded to %s'; done
```

Comment:
nit: commit message should explain why this is needed.

Reply:
Added explanation to commit message.